core/templates/dev/head/pages/story-viewer-page/story-viewer-page.controller.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the story viewer.
+ */
+
+require(
+  'components/common-layout-directives/common-elements/' +
+  'background-banner.directive.ts');
+require(
+  'pages/story-viewer-page/navbar-breadcrumb/' +
+  'story-viewer-navbar-breadcrumb.directive.ts');
+require(
+  'pages/story-viewer-page/chapters-list/' +
+  'story-viewer-chapters-list.directive.ts');
+
+require('domain/story_viewer/ReadOnlyStoryNodeObjectFactory.ts');
+require('domain/story_viewer/StoryViewerBackendApiService.ts');
+require('domain/story_viewer/StoryPlaythroughObjectFactory.ts');
+
+require('services/AlertsService.ts');
+require('services/PageTitleService.ts');
+require('services/contextual/UrlService.ts');
+
+oppia.controller('StoryViewer', [
```

Comment:
Add a component directive instead of a standalone controller.

Reply:
Done